src/concepts/segments.md

```diff
@@ -0,0 +1,105 @@
+# Segments
+
+A segment is a group of clients who fit a set of criteria at a point in time. 
+The set of criteria itself can also be referred to as a "segment".
+
+Typically you'll use segments to gain more insight into what is going on, by asking 
+"Regarding the thing I'm interested in, 
+do users in different segments behave differently, 
+and what insights does this give me into the users and the product?" 
+For example, "In this experiment, how do new users react to this feature, 
+and how does this differ from established users?". 
+Or "DAU moved dramatically - 
+is this restricted to users in this particular country (i.e. a segment) 
+where there's an event happening, which would raise our suspicions,
+or is it global and therefore not solely due to that event?"
+
+## Versioning
+
+We are building out our library of segments, 
+and we want room to iterate to improve them in the future. 
+So please quote segments' versions with their names, e.g. "regular users v1"
+so that your communication is forwards compatible.
+
+## Current segments
+
+### Regular users v1, Semi-regular users v1, New/irregular users v1
+
+This is a set of three segments. 
+On a given day, every client falls into exactly one of these segments.
+Each client's segment is stored in `telemetry.clients_last_seen.segment_regular_users_v1`.
+
+
+*Regular users v1* is defined as 
+clients who browsed >=5 URIs on at least _one_ of the previous 6 days, and browsed >=5 URIs on _at least two_ days in each of the three weeks before that. 
+Observationally, on any given day this segment seems to contain a large fraction of our users 
+and has exceptionally high retention.
+
+*New/irregular users v1* is defined as 
+clients who did not browse >=5 URIs on any of the previous 6 days, and browsed >=5 URIs on _less than two_ days in each of the three weeks before that.
+This is a smaller segment of our users and has low retention 
+(though "activation" is likely a more relevant word than "retention" for many of these clients).
+
+*Semi-regular users v1* contains the other clients.  
+This seems to be a smaller segment of our users, and has relatively high retention.
```

Comment:
I thought you were going to add "We do not yet know what proportion of users in this segment stay in this segment for an extended period, and what proportion are in transition between other segments." here?